apps/server/src/modules/course/repo/coursegroup.repo.integration.spec.ts

```diff
@@ -150,34 +150,60 @@ describe('course group repo', () => {
 		});
 	});
 
-	describe('update courseGroup', () => {
+	describe('removeUserReference', () => {
 		describe('when user is existing', () => {
 			const setup = async () => {
 				// Arrange
 				const course = courseEntityFactory.build();
 				const courseGroup1 = courseGroupEntityFactory.studentsWithId(3).build({ course });
-				const courseGroup2 = courseGroupEntityFactory.build({ course });
+				const courseGroup2 = courseGroupEntityFactory.studentsWithId(2).build({ course });
 				const userId = courseGroup1.students[0].id;
 				await em.persistAndFlush([courseGroup1, courseGroup2]);
+				em.clear();
 
 				return {
 					userId,
 					courseGroup1,
+					courseGroup2,
 				};
 			};
 
-			it('should update courseGroup without userId', async () => {
+			it('should call nativeUpdate', async () => {
+				const { userId } = await setup();
+
+				jest.spyOn(em, 'nativeUpdate');
```

Comment:
Same spy in integration test. Please remove and test user inside the db.

Reply:
was discussed above